### PR TITLE
Hotfix/secondary skip nav timing issue

### PR DESCRIPTION
**Pull request by gileswells**
### Description
Top level documentation pages (eg. /explore/benefits) would not properly take focus when using the skip nav. This resolves that issue.

### Process
- [ ] Tests added or updated for change
- [ ] Docs updated
- [ ] Accessibility concerns have been considered and addressed

### CODE REVIEW

File: src/utils/clickHandlers.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ export const onHashAnchorClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
   // This provides for focus to be applied when clicking a /hash#link
   const id: string = e.currentTarget.href.split('#')?.[1];
   if (id) {
-    document.getElementById(id)?.focus();
+    setTimeout(() => {
+      document.getElementById(id)!.focus();
```
Comment (anthony-kopczyk) [inline src/utils/clickHandlers.ts:8]: Is the change from `?` to `!` purposeful? I don't think the eslinter likes `!`

**Review by anthony-kopczyk — COMMENTED** [review]
Just one question.

File: src/utils/clickHandlers.ts (hunk 1 above)
Comment (gileswells) [reply src/utils/clickHandlers.ts]: That was a goof from my first attempts to fix it. That's changed back now.

**Review by anthony-kopczyk — APPROVED** [review]
Looks good to me!

**Pull request merged**